features/support/env.rb

```diff
@@ -6,6 +6,8 @@
 require File.expand_path("../../../spec/shared_helper", __FILE__)
 
 require 'cucumber/rails'
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.